test/integration/spec/connect.js

```diff
@@ -1257,11 +1258,11 @@ describe('connect', function() {
           }[switchOffParticipant];
 
```

Comment:
@makarandp0 Can you temporarily log the "switchedOn" and "switchedOff" events here and run the CircleCI tests once more? This way we can find out if we get back to back events:
```js
Object.values(switched).forEach(({ remoteVideoTrack }) => {
  ['switchedOff', 'switchedOn'].forEach(event => {
    remoteVideoTrack.on(event, () => console.log(`${event}: ${remoteVideoTrack.name}`));
  });
});
```